### DISCUSSION TITLE
Why does my setup seem to be missing features?

### DISCUSSION THREAD

**Discussion by ghost** [TomBursch/kitchenowl#464] [Q&A]
Hi there!

From some of the issues (e.g. [https://github.com/TomBursch/kitchenowl/issues/51](https://github.com/TomBursch/kitchenowl/issues/51), [https://github.com/TomBursch/kitchenowl/issues/331](https://github.com/TomBursch/kitchenowl/issues/331), [https://github.com/TomBursch/kitchenowl/issues/358](https://github.com/TomBursch/kitchenowl/issues/358) or the [database model](https://dbdiagram.io/d/KitchenOwl-Data-Structure-Proposal-659bed9dac844320ae74edca) )  I'm seeing features being mentioned that I cannot find in my setup.
I have exactly one shopping list, I cannot edit or change the order of categories, I cannot edit a household (e.g. re-assign a language) and so forth.
Are some of the mentioned features only available in the cloud storage or older versions of the software? Or did I maybe misconfigure something?

Best regards

**Comment by TomBursch** [comment]
The self hosted version has all features and is equal to the cloud version. Did you create the household? Only household admins can edit some household settings (Profile -> settings -> household -> settings) if you cannot see that option ask the creator of your household to promote you to admin.

**Reply by ghost** [reply]
Oh wow, today I learned the difference between server admin and household admin. I mean, I was quite sure that I promoted my user to household admin just before I deleted the initial admin account, but it seems I was misremembering.
Now a couple of options appeared, thanks a lot!
